pages.es/common/az-webapp.md

```diff
@@ -0,0 +1,21 @@
+# az webapp
+
+> Administra aplicaciones web alojadas en servicios de nube de Azure.
+> Parte de `azure-cli`.
+> Más información: <https://learn.microsoft.com/es-es/cli/azure/webapp>.
```

Comment:
```suggestion
> Más información: <https://learn.microsoft.com/cli/azure/webapp>.
```
We don't use l10n in Microsoft sites because based on your system's language settings or browser's locale it automatically redirects you to the translated version while respecting your language preference.